src/components/learnCollection/Card.tsx

```diff
@@ -134,6 +135,8 @@ const Card: React.FC<CardProps> = ({
           index={index}
           length={length}
         />
+        {user !== null ? <img src={user.photoUrl} alt="user profile photo" /> : null
```

Comment:
Can we add user !== user.photoUrl too? It should catch if someone does not have a photo